js/ctrls/modal.js

```diff
@@ -79,15 +79,15 @@ angular
     parse: function() {
       return _
         .chain(this.uris.trim().split(/\r?\n/g))
-        .map(function(d) { 
+        .map(function(d) {
           return _(d)
-            .replace(/("[^"]*")/g, function(c) {
-              return c.replace('%','%25').replace(' ','%20');
+            .replace(/["'][^"]*['"]/g, function(c) {
```

Comment:
Shouldn’t this be /[“‘][^”’][“‘]/g ?

Reply:
@weirongxu

Reply:
Yes, I forget this part
I'll be updated immediately